### DISCUSSION TITLE
GIthub Darkmode breaks on overlapping element

### DISCUSSION THREAD

**Discussion by gauthamp10** [community/community#2902] [Other Feature Feedback, Questions, & Ideas] [answered]
Darkmode breaks on overlapping element or extensions.

![JBdRCwP](https://user-images.githubusercontent.com/31800012/101667115-f373c500-3a74-11eb-84c4-1ab68d1fda4e.png)

Browser: Brave Version 1.17.75 Chromium: 87.0.4280.88 (Official Build) (64-bit)

PS: the extension used in the screenshot is a Vomnibar for [vimium](https://chrome.google.com/webstore/detail/vimium/dbepggeogbaibhgnhhndojpepiihcmeb/related?hl=en) browser extension . This doesn't occur on other dark modes in sites like Youtube though.

**Comment by evi-liu** [comment, marked as answer]
Hi there! This forum is for Discussions feedback. Do you mind posting this feedback on darkmode in https://github.community/c/github-help/dark-mode-beta/65?

**Reply by gauthamp10** [reply]
Thank you.
